admin/bsf-analytics/class-bsf-analytics-stats.php

```diff
@@ -105,7 +105,7 @@ private function get_default_stats() {
 		/**
 		 * Get installed PHP version.
 		 *
-		 * @return float PHP version.
+		 * @return string PHP version.
```

Comment:
Please test type changes very carefully, very often these things break on the front end